### PR TITLE
API final project 

### PR DESCRIPTION
**Pull request by michaelfelsenthal**

### CODE REVIEW

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 1]
```diff
@@ -1 +1,104 @@
 
+
+# General Putnam Diner API
+
+## Introduction 
+
+The owners of General Putnam Diner are creating an app for their customers to make touchless orders . This will enable customers to make takeout orders.
+
+In the future, the owners would like to place the app on a tablet so customers can order from their seat. 
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:10]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 2]
```diff
@@ -1 +1,104 @@
...
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:11]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 3]
```diff
@@ -1 +1,104 @@
...
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:12]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 4]
```diff
@@ -1 +1,104 @@
...
+The application will interface with the server side ordering system in the kitchen which prints out orders for the cooks. The API is the interface between the app and the server. 
+
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:15]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 5]
```diff
@@ -1 +1,104 @@
...
+## Introduction 
+
+The owners of General Putnam Diner are creating an app for their customers to make touchless orders . This will enable customers to make takeout orders.
+
+In the future, the owners would like to place the app on a tablet so customers can order from their seat. 
+
+
+
+The application will interface with the server side ordering system in the kitchen which prints out orders for the cooks. The API is the interface between the app and the server. 
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:14]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 6]
```diff
@@ -1 +1,104 @@
...
+
+
+The diner hired ACME APP company to provide them with this app. The developers have been assigned to provide a proof of concept and finished app. Once approved, the General Putnam Diner will sign off and become a customer.
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:18]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 7]
```diff
@@ -1 +1,104 @@
...
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:19]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 8]
```diff
@@ -1 +1,104 @@
...
+
+## Error Codes
+
+The General Putnam Diner API follows codes based on the success or error of the order.
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:24]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 9]
```diff
@@ -1 +1,104 @@
...
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:25]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 10]
```diff
@@ -1 +1,104 @@
...
+
+Code Type| Code Description 
+--- | ---
+200 OK | The order was placed successfully.
+401 Order Incomplete | A section of the order was not filled out. Examples can include not filling out "pattyType" or not checking a selection for "drink."
+405 Missing Information | The customer's contact information is missing. A cell phone number and email is required to complete an order. 
+
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:33]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 11]
```diff
@@ -1 +1,104 @@
...
+## Resources
+
+These are the resources for the General Putnam API:
+
+
+
+[GET Reference Guide](https://github.com/Laura-Novich-OBW/student-showcase/blob/main/student-work/michael-felsenthal/api-final-project/GET%20table.md).
+
+
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:44]:
```suggestion
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 12]
```diff
@@ -1 +1,104 @@
...
+[Post Reference Guide](https://github.com/Laura-Novich-OBW/student-showcase/blob/main/student-work/michael-felsenthal/api-final-project/POST.md).
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:45]:
```suggestion
* [Post Reference Guide](https://github.com/Laura-Novich-OBW/student-showcase/blob/main/student-work/michael-felsenthal/api-final-project/POST.md).
```

File: student-work/michael-felsenthal/api-final-project/readme.md [hunk 13]
```diff
@@ -1 +1,104 @@
...
+
+
+## API Work Flow 
+
+```mermaid
+
+graph TD;
+
+A[Place order]-->B[select burger meal];
+
```
Comment (Laura-Novich-OBW) [inline student-work/michael-felsenthal/api-final-project/readme.md:56]:
```suggestion
```

**Review by Laura-Novich-OBW — COMMENTED** [review]
please remove the empty lines which are all over this file.
If you want some white space one or two lines is fine.
In your code sample, however, there cannot be any empty lines.

**Comment by Laura-Novich-OBW** [comment]
@michaelfelsenthal please make the suggested changes plus the changes in my comments.

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
please remove the empty lines which are all over this file.
If you want some white space one or two lines is fine.
In your code sample, however, there cannot be any empty lines.

**Comment by michaelfelsenthal** [comment]
All changes implemented.

**Pull request closed without merging**